test/TestOrchestrations.cs

```diff
@@ -80,6 +80,33 @@ public static string SayHelloInline([OrchestrationTrigger] DurableOrchestrationC
             return currentValue;
         }
 
+        public static async Task<int> InfiniteCounter([OrchestrationTrigger] DurableOrchestrationContext ctx)
+        {
+            int currentValue = ctx.GetInput<int>();
+
+            bool done = false;
+            while (!done)
+            {
+                System.Console.WriteLine($@"Current value is {currentValue}");

```

Comment:
Let's leave out `Console.WriteLine` from the test code.

Reply:
Sure thing. I couldn't figure out how to get *any* output to the Test harness from w/in the actual execution :(